inst/tests/tests.Rraw

```diff
@@ -10059,7 +10059,7 @@ test(1658.56, fwrite(data.table(exp(1) - pi*1i)), output='2.718[0-9]*-3.141[0-9]
 DT = data.table(a=1:3, b=list(1:4, c(3.14, 100e10), c(3i,4i,5i)))
 test(1658.57, fwrite(DT), output='0+3i|0+4i|0+5i')
 DT[ , b := c(1i, -1-1i, NA_complex_)]
-test(1658.58, fwrite(DT), output='a,b\n1,0\\+1i\n2,-1-1i\n3,$')
+test(1658.58, fwrite(DT), output='a,b\n1,0+1i\n2,-1-1i\n3,\n[1] ""')
```

Comment:
```suggestion
test(1658.58, invisible(fwrite(DT)), output='a,b\n1,0\\+1i\n2,-1-1i\n3,$')
```

Reply:
this seems to be failing the test 
```
 Test 1658.58 did not produce correct output:
  Expected: <<a,b\n1,0\\+1i\n2,-1-1i\n3,$>>
  Observed: <<a,b\n1,0+1i\n2,-1-1i\n3,\n[1] "">>
  ```
  and why does this test breaks to begin with?, its not like this PR changes anything related to output.

Reply:
Oh, right, because `output=` test always runs explicit `print()` under the hood. So to reproduce locally you can run `print(fwrite(...))` and you'll see the `[1] ""` part.

That said, we don't need to test for that. I've updated the suggested edit.